examples/notebook/dynas/ResNet50_Quantiation_Search_Supernet_NAS.ipynb

```diff
@@ -0,0 +1,481 @@
+      "2023-09-05 14:10:22 [INFO] The cfg.tuning.exit_policy.performance_only is: True\n",
+      "2023-09-05 14:10:22 [INFO] Attention Blocks: 0\n",
+      "2023-09-05 14:10:22 [INFO] FFN Blocks: 0\n",
+      "2023-09-05 14:10:22 [INFO] Pass query framework capability elapsed time: 197.49 ms\n",
+      "2023-09-05 14:10:22 [INFO] Adaptor has 2 recipes.\n",
+      "2023-09-05 14:10:22 [INFO] 0 recipes specified by user.\n",
+      "2023-09-05 14:10:22 [INFO] 0 recipes require future tuning.\n",
+      "2023-09-05 14:10:22 [INFO] Neither evaluation function nor metric is defined. Generate a quantized model with default quantization configuration.\n",
+      "2023-09-05 14:10:22 [INFO] Force setting 'tuning.exit_policy.performance_only = True'.\n",
+      "2023-09-05 14:10:23 [WARNING] Please note that calibration sampling size 100 isn't divisible exactly by batch size 64. So the real sampling size is 128.\n",
+      "2023-09-05 14:10:32 [INFO] |*********Mixed Precision Statistics********|\n",
+      "2023-09-05 14:10:32 [INFO] +---------------------+-------+------+------+\n",
+      "2023-09-05 14:10:32 [INFO] |       Op Type       | Total | INT8 | FP32 |\n",
+      "2023-09-05 14:10:32 [INFO] +---------------------+-------+------+------+\n",
+      "2023-09-05 14:10:32 [INFO] |      ConvReLU2d     |   29  |  10  |  19  |\n",
+      "2023-09-05 14:10:32 [INFO] | quantize_per_tensor |   17  |  17  |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |         add         |   1   |  1   |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |      dequantize     |   15  |  15  |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |      MaxPool2d      |   1   |  1   |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |        Conv2d       |   17  |  10  |  7   |\n",
+      "2023-09-05 14:10:32 [INFO] |       add_relu      |   13  |  13  |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |      AvgPool2d      |   1   |  1   |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |         mean        |   2   |  2   |  0   |\n",
+      "2023-09-05 14:10:32 [INFO] |        Linear       |   1   |  0   |  1   |\n",
+      "2023-09-05 14:10:32 [INFO] +---------------------+-------+------+------+\n",
+      "2023-09-05 14:10:32 [INFO] Pass quantize model elapsed time: 9604.32 ms\n",
+      "2023-09-05 14:10:32 [INFO] Save tuning history to /localdisk/maciej/code/neural-compressor/examples/notebook/dynas/nc_workspace/2023-09-05_14-07-22/./history.snapshot.\n",
+      "2023-09-05 14:10:32 [INFO] Specified timeout or max trials is reached! Found a quantized model which meet accuracy goal. Exit.\n",
+      "2023-09-05 14:10:32 [INFO] Save deploy yaml to /localdisk/maciej/code/neural-compressor/examples/notebook/dynas/nc_workspace/2023-09-05_14-07-22/deploy.yaml\n"
+     ]
+    },
+    {
+     "ename": "KeyboardInterrupt",
```

Comment:
Seems notebook haven't run successfully, I think it's better to have a successful run.

Reply:
Oops! Good catch - commit was created without staging changes in the notebook. Thanks!

Reply:
No problem.